tests/after_cluster_deploy_sanity/test_after_cluster_deploy_sanity.py

```diff
@@ -164,3 +164,10 @@ def test_csv_health(admin_client, hco_namespace):
         timeout=TIMEOUT_5MIN,
         stop_status="fakestatus",  # to bypass intermittent FAILED status that is not permanent.
     )
+
+
+@pytest.mark.cluster_health_check
```

Comment:
Question: Do we want to take an action based on the outcome of this?

Reply:
Yes. Devops would probably decide on rebuilding the cluster or allocate them for tests that does not require migration.